usr/sbin/p11sak/p11sak.c

```diff
@@ -912,6 +960,73 @@ static void set_bool_attr_from_string(CK_ATTRIBUTE *attr, char attr_char)
     }
 }
 
+/**
+ * @brief Set the custom attrs from string object
+ * 
+ * @param attr The attribute array
+ * @param attr_char The string from the user
+ * @param count Index of current attribute in the attribute array
+ */
+static CK_RV set_custom_attrs_from_string(char *attr_char, CK_ATTRIBUTE *attr, CK_ULONG *count) 
+{
+    CK_RV rc = CKR_OK;
+    int i = 0, j = 0;
+    char substr[10];
+
+    // iterate over the single custom attributes
+    while (i < (int)strlen(attr_char))
```

Comment:
Personally I would change the `i` to an unsigned value.  In this scenario, a string longer than the maximal positive integer would be skipped.  For command line arguments, this might be irrelevant, but if `attr_char` can ever come from a different source...

Reply:
Done